src/components/templates/Header.jsx

```diff
@@ -53,12 +53,12 @@ const Header = () => {
           <div
             className={
               dropDown
```

Comment:
While adding the transition/animation we don't need to manipulate all the classes, just manipulate classes that are needed. The classes that are in both conditions are those before the condition. With this, there will be no duplications.